frontend/ashoka-the-great-explorer/style.css

```
@@ -0,0 +1,646 @@
+    box-shadow: 0 4px 15px rgba(16, 185, 129, 0.2);
+}
+
+.edict-display-card {
+    padding: 24px;
+    border-radius: 18px;
+    background: rgba(0, 0, 0, 0.2);
+    border: 1px solid var(--ashoka-border);
+    animation: fadeInAnim 0.3s ease;
+}
+
+@keyframes fadeInAnim {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeInAnim" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)